static/scss/flex/ttt.scss

```diff
@@ -62,4 +54,12 @@ i.fa.fa-chevron-down {
   line-height: 1.8rem;
   color: black;
   text-indent: -.5rem;
-}
+
+  i.fa.fa-chevron-right {
```

Comment:
Moved this under the .expand-label, since it was also applying to the right-chevron in the footer nav. 

Oh, cascade, how we love you.